DocumentConversion/ConvertToPDFX/ConvertToPDFX.cpp

```diff
@@ -4,7 +4,7 @@
 // ConvertToPDFX converts the input PDF to a PDF/X compliant PDF.
 //
 // Command-line:  <input-pdf> <convert-option>       (all parameters are optional moving from left to right)
-//        where convert-option is 'PDFX4', 'PDFX1a2001', 'PDFX32003'
+//        where convert-option is 'PDFX4', 'PDFX6', 'PDFX1a2001',or  'PDFX32003'
```

Comment:
lets add a space after `or`  so its not right next to the comma.

Reply:
fixed